### ISSUE TITLE
tab autocomplete is not working

### ISSUE THREAD

**Issue by vp70** [jimradford/superputty#573]
I noticed this issue few times.
 When i undock my laptop i will loose connection and my putty session.
Then I would do right click on a tab with server name on it and select "restart session".
After session successfully restarted i would start typing name of a file and click tab on my keyboard but auto complete would not work. 
However if i close this tab and open another tab , login, tab autocomplete works again on new tab tab I just opened.
So in summary somewhere after "Restart Session" performed we loose ability to do Tab autocomplete on restarted tab.

-VP

**Comment by jimradford** [comment]
Thanks for the report, I'll see if I can reproduce the behavior you are experiencing.

**Comment by Ehekatl** [comment]
+1 on this
